vividus-plugin-web-app/src/main/resources/properties/profile/saucelabs/web/profile.properties

```diff
@@ -0,0 +1,4 @@
+selenium.grid.capabilities.sauce\:options.idleTimeout=${selenium.grid.idle-timeout}
+selenium.grid.capabilities.sauce\:options.seleniumVersion=${selenium.grid.selenium-version}
+selenium.grid.capabilities.sauce\:options.screenResolution=${selenium.grid.screen-resolution}
+selenium.grid.capabilities.sauce\:options.commandTimeout=180
```

Comment:
Why the command timeout changed?

Reply:
sauce labs specific options should be places under sauce\:options as per documentation, this prop doesn't influence other grids

Reply:
it was 300 now it's 180

Reply:
It was 300 for tables and phones (in appium this props should be newCommandTimeout), for web it was 180